pax-web-extender-war/src/main/java/org/ops4j/pax/web/extender/war/internal/model/WebAppServlet.java

```diff
@@ -124,25 +120,6 @@ public void setServletClassName(final String servletClassName) {
 	}
 
 	/**
-	 * Getter.
-	 * 
-	 * @return servletClass
-	 */
-	public Class<? extends Servlet> getServletClass() {
-		return servletClass;
-	}
-
-	/**
-	 * Setter.
-	 * 
-	 * @param servletClass
-	 *            value to set.
-	 */
-	public void setServletClass(final Class<? extends Servlet> servletClass) {
-		this.servletClass = servletClass;
-	}
-
-	/**
 	 * Returns the aliases associated with this servlet. If there are no
 	 * associated aliases an empty array is returned.
 	 * 
```

Comment:
Nope, this is unfortunately not the way we're going to handle this. ServletClasses are essential to have injection via Pax CDI possible.